api/src/main/java/uk/gov/dhsc/htbhf/claimant/factory/VerificationResultFactory.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.dhsc.htbhf.claimant.factory;
+
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.model.VerificationResult;
+import uk.gov.dhsc.htbhf.eligibility.model.CombinedIdentityAndEligibilityResponse;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.apache.commons.collections4.CollectionUtils.containsAny;
+
+public class VerificationResultFactory {
+
+    /**
+     * Creates a {@link VerificationResult} from a given {@link Claimant} and {@link CombinedIdentityAndEligibilityResponse}.
+     * @param claimant claimant details
+     * @param identityAndEligibilityResponse response from eligibility serivce
+     * @return a {@link VerificationResult}
+     */
+    public static VerificationResult buildVerificationResult(Claimant claimant, CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse) {
```

Comment:
All of this code is lifted and shifted from ClaimResultDTO, same goes for the unit test.